requirements/edx-sandbox/py35.txt

```diff
@@ -4,25 +4,25 @@
 #
 #    make upgrade
 #
-common/lib/sandbox-packages  # via -r requirements/edx-sandbox/py35.in
-common/lib/symmath  # via -r requirements/edx-sandbox/py35.in
+-e file:///edx/app/edxapp/edx-platform/common/lib/sandbox-packages  # via -r requirements/edx-sandbox/py35.in
```

Comment:
This needs to be fixed, I think there is a script we run as a part of the `upgrade` make target that needs to be run on this file as well.

Reply:
In Makefile, this command, `scripts/post-pip-compile.sh $(REQ_FILES:=.txt)`, should be fixing this but is not for some reason.  This needs to be reviewed.